scripts/download.py

```diff
@@ -42,7 +42,7 @@ def download_from_hub(
             " https://huggingface.co/settings/tokens"
         )
 
-    download_files = ["tokenizer*", "generation_config.json"]
+    download_files = ["tokenizer*", "*config.json"]
```

Comment:
Lets be explicit instead of using *. We support many weights and there's no structure about what they can include